tests/final_chain_test.cpp

```diff
@@ -277,6 +278,36 @@ TEST_F(FinalChainTest, nonce_skipping) {
   advance({std::make_shared<Transaction>(5, 13, 0, TRX_GAS, dev::bytes(), key.secret(), key.address())});
 }
 
+TEST_F(FinalChainTest, initial_validators) {
+  const dev::KeyPair key = dev::KeyPair::create();
+  const std::vector<dev::KeyPair> validator_keys = {dev::KeyPair::create(), dev::KeyPair::create(),
+                                                    dev::KeyPair::create()};
+  cfg.state.genesis_balances = {};
+  cfg.state.dpos.emplace();
+  cfg.state.dpos->eligibility_balance_threshold = 100000;
+  cfg.state.dpos->vote_eligibility_balance_step = 10000;
+  cfg.state.dpos->maximum_stake = 10000000;
+  cfg.state.dpos->minimum_deposit = 1000;
+  cfg.state.dpos->eligibility_balance_threshold = 1000;
+
+  for (const auto& vk : validator_keys) {
+    state_api::ValidatorInfo validator{vk.address(), key.address(), 0, "", "", {}};
+    validator.delegations.emplace(key.address(), cfg.state.dpos->maximum_stake);
+    validator.delegations.emplace(validator_keys.front().address(), cfg.state.dpos->maximum_stake);
+    cfg.state.dpos->initial_validators.emplace_back(validator);
+  }
+
+  cfg.state.genesis_balances[key.address()] = 1000000000;
+  cfg.state.genesis_balances[validator_keys.front().address()] = 1000000000;
+  init();
+  auto total_votes = SUT->dpos_eligible_total_vote_count(SUT->last_block_number());
+  for (const auto& vk : validator_keys) {
+    auto address_votes = SUT->dpos_eligible_vote_count(SUT->last_block_number(), vk.address());
+    EXPECT_EQ(total_votes / validator_keys.size(), address_votes);
```

Comment:
probably not a good idea to use division in uint result comparison. Let's say someone changes the code above so 
```
total_votes  = 5
validator_keys = 3 
and the divisionr esult = 1
```